docs/source/models/vlm.rst

```diff
@@ -9,10 +9,6 @@ This document shows you how to run and serve these models using vLLM.
 .. important::
     We are actively iterating on VLM support. Expect breaking changes to VLM usage and development in upcoming releases without prior deprecation.
 
-    Currently, the support for vision language models on vLLM has the following limitations:
-
-    * Only single image input is supported per text prompt.
```

Comment:
Perhaps we can add a line to say users can refer to the supported model section to see which models actually work with multiple images input.

Reply:
I'm thinking of updating the docs to include an example of multi-image input, but unfortunately LLaVA-1.5 isn't really a good candidate for showcasing this. What do you think of switching to Phi-3.5-Vision for the example?

Reply:
Yea I actually updated the vision api client example with phi3.5v instruct, so feel free to just use that one.

Reply:
Ok let me try to do it in this PR then since it already exists.

Reply:
Done, please take a look.

Reply:
Overall looks good to me. Feel free to merge whenever you find it ready!